semgrep/semgrep/semgrep_main.py

```diff
@@ -288,16 +293,20 @@ def main(
     num_findings = sum(len(v) for v in rule_matches_by_rule.values())
     stats_line = f"ran {len(filtered_rules)} rules on {len(all_targets)} files: {num_findings} findings"
 
-    project_hash = None
     try:
         project_url = sub_check_output(
             ["git", "ls-remote", "--get-url"],
             encoding="utf-8",
             stderr=subprocess.DEVNULL,
         )
-        project_hash = hashlib.sha256(project_url.encode()).hexdigest()
     except Exception as e:
-        logger.debug(f"Failed to generate project hash: {e}")
+        logger.debug(f"Failed to get project url from 'git ls-remote': {e}")
+        # add \n to match urls from git ls-remote (backwards compatability)
+        project_url = manually_search_file(".git/config", ".com", "\n")
```

Comment:
Probably also want to put this in a try-except since file reading can potentially fail (doesn't exits, wrong permissions, wrong format, etc.)

Reply:
fixed